.github/CODEOWNERS

```diff
@@ -159,6 +159,7 @@
 
 # PRLabel: %Open AI
 /sdk/openai/azure-ai-openai/                         @brandom-msft @jpalvarezl @mssfang
+/sdk/openai/azure-ai-openai-assistants/              @brandom-msft @jpalvarezl @mssfang
```

Comment:
This will not get a PR label.  The comment on L160 applies only to the first line after it.   You'll probably want to add a copy above this entry.

We should also have Service Attention rule for this.  Right now, adding the `Service Attention` label will do nothing.

Reply:
Also `brandom-msft` needs to make the Microsoft and Azure organization memberships public to be valid.

Reply:
Thanks Jesse

Reply:
@brandom-msft Can you follow https://eng.ms/docs/products/azure-developer-experience/onboard/access to make public to be valid.